src/lib/builders/select/index.ts

```diff
@@ -96,6 +77,38 @@ export function createSelect(args?: CreateSelectArgs) {
 		}
 	);
 
+	const trigger = elementMultiDerived(
```

Comment:
Now that I think about it, does it make sense for trigger to be multi derived? I'd say no, right?